### PR TITLE
[installer] Fix registry-facade ClusterRoleBinding name

### PR DESCRIPTION
**Pull request by aledbf**
## Description

The ClusterRoleBinding must include the namespace to allow multiple installations

## Release Notes
```release-note
[installer] Fix registry-facade ClusterRoleBinding name
```

### CODE REVIEW

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please ask for approval from **aledbf** after the PR has been reviewed.

*No associated issue*. Update pull-request body to add a reference to an issue, or get approval with `/approve no-issue`

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

<details open>
Needs approval from an approver in each of these files:

- **[installer/pkg/components/registry-facade/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/installer/pkg/components/registry-facade/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by kylos101 — APPROVED** [review]
/LGTM

**Comment by roboquat** [comment]
LGTM label has been added.  <details>Git tree hash: 8e1bf5b4971ca6c051ba3e8cbfe0d4613d05c6b7</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/7670#pullrequestreview-856020680" title="LGTM">kylos101</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please ask for approval from **aledbf** after the PR has been reviewed.

*No associated issue*. Update pull-request body to add a reference to an issue, or get approval with `/approve no-issue`

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

<details open>
Needs approval from an approver in each of these files:

- **[installer/pkg/components/registry-facade/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/installer/pkg/components/registry-facade/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/7670#issuecomment-1015871295" title="Approved">kylos101</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please ask for approval from **aledbf** after the PR has been reviewed.

*No associated issue*. Update pull-request body to add a reference to an issue, or get approval with `/approve no-issue`

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

<details open>
Needs approval from an approver in each of these files:

- **[installer/pkg/components/registry-facade/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/installer/pkg/components/registry-facade/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/7670#issuecomment-1015871752" title="Approved">aledbf</a>*, *<a href="https://github.com/gitpod-io/gitpod/pull/7670#issuecomment-1015871295" title="Approved">kylos101</a>*

Associated issue requirement bypassed by: *<a href="https://github.com/gitpod-io/gitpod/pull/7670#issuecomment-1015871752" title="Approved">aledbf</a>*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[installer/pkg/components/registry-facade/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/installer/pkg/components/registry-facade/OWNERS)~~ [aledbf]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
